README.md

```diff
@@ -34,104 +34,55 @@ Getting Started
 
 Ketrew is very flexible and hence may seem difficult to understand and setup at
 first.
-Let's get a minimal workflow running.
+Let's get a minimal setup ready and a workflow running on it.
```

Comment:
Might be worth mentioning how minimal and not-production ready this is (and point to secotrec repo with more info there eventually on getting a production-ready setup going?)